### DISCUSSION TITLE
CI timing out without any timeout config

### DISCUSSION THREAD

**Discussion by turnsy** [community/community#130792] [Actions]
### Select Topic Area

Question

### Body

Hello all,

One of my CI jobs is failing, because one of it's actions is timing out(?), then getting marked as cancelled. This occurs now because I added a bunch of larger files to the repository, and this specific action builds & ships a docker image containing those files. So my assumption is it's timing out, but `timeout-minutes` is not set anywhere in the yml files.

It's also worth noting that no logs are shown from the cancelled job, but when I watch it live it fails at different spots confirming that it's not an issue with the job itself. Further, the job runs in about 5 minutes when successful.

I have tried:
- `fail-fast = false`
- Setting `timeout-minutes` to 20 minutes

Am I missing anything obvious here?

**Comment by HemalathaT2** [comment]
Check the runner's timeout
Even if you haven't set timeout-minutes in your YAML file, the runner itself might have a default timeout. Check your CI provider's documentation to see if there's a default timeout for runners. For example, in GitLab CI/CD, the default timeout is 60 minutes.

**Comment by joshjohanning** [comment]
Is a runner picking up the job, or is it still in a queue?

**Comment by turnsy** [comment]
For what it's worth, the issue was with the base image not having enough memory, and breaking that way- though no error was shown in actions, or a log to read through. It was only marked as cancelled.

**Reply by grandmas-favorite** [reply]
Thanks for coming back to share your solution @turnsy! This will surely help others that experience the same issue.

**Reply by turnsy** [reply]
Of course! To clarify, if you see a silent fail/cancel without reason and you're building docker images, try sizing up the resources allocated to it.
